docs/UserGuide.md

```diff
@@ -49,41 +49,64 @@ See [How do I move a file?](#q-how-do-i-move-a-file) for help.
 
 2. Open a terminal or your computer.
 <box type="info" seamless>
-    <ul>
-        <li><strong>For Mac users:</strong> Open the Terminal app on your Mac</li>
-        <li><strong>For Windows users:</strong> Press the Windows key on your keyboard, type `powershell` into the 
-                    dialog box that opens and press "Enter" on your keyboard to open up a terminal window.</li>
+
+        <li><strong>For Mac users:</strong> open the Terminal app on your Mac</li>
+        <li><strong>For Window users:</strong> press the windows key, type `powershell` and press "Enter" on your keyboard</li>
+        <li><strong>See <a href="#faq">How do I open a terminal?</a> for more information.</li>
     </ul>
-    See <a href="#faq">How do I open a terminal?</a> for more information.
 </box>
 
 3. Use `cd` command in the terminal to navigate to the same location as `hrmate.jar` in step 1. See [How do I navigate files in terminal?](#faq)
 
-4. Ensure you have Java `11` or above installed in your Computer. This can be done by typing `java -version` in your terminal and pressing "Enter".
- - See <a href="#faq">How do I download Java 11?</a> if Java is not updated or if the terminal does not recognise the `java` input. 
+4. Ensure you have `Java 11` or above installed on your Computer. This can be done by typing `java -version` in your terminal.
+ - See <a href="#faq">How do I download Java 11?</a> if Java is not updated or if the terminal does not recognize the `java` input. 
```

Comment:
I think that the pressing enter is important, and recognize is not british english spelling